src/repositories/__test__/qr.repo.integration.test.ts

```
@@ -0,0 +1,147 @@
+      const token = generateRandomToken();
+      const ip = '127.0.0.1';
+      const userAgent = 'test-agent';
+
+      await repo.createQRLoginToken(token, TEST_DATA.USER_ID, ip, userAgent);
+      const foundToken = await repo.findQRLoginToken(token);
+
+      expect(foundToken).not.toBeNull();
+      expect(foundToken?.token).toBe(token);
+      expect(foundToken?.is_used).toBe(false);
+      if (foundToken) {  
+        expect(new Date(foundToken.expires_at).getTime()).toBeGreaterThan(new Date(foundToken.created_at).getTime());  
```

ESLint

[error] Avoid calling `expect` conditionally`

(jest/no-conditional-expect)